docs/troubleshooting/missing-dotmount-files.rst

```diff
@@ -0,0 +1,79 @@
+Inaccessible "/var/lib/scylla" and "/var/lib/systemd/coredump" after ScyllaDB upgrade 
+======================================================================================
+
+Problem
+^^^^^^^
+When you reboot the machine after a ScyllaDB upgrade, you cannot access data directories under ``/var/lib/scylla``, and 
+coredump saves to ``rootfs``.
+
+
+The problem may occur when you upgrade ScylaDB Open Source 4.6 or later to a version of ScyllaDB Enterprise if
+the ``/etc/systemd/system/var-lib-scylla.mount`` and ``/etc/systemd/system/var-lib-systemd-coredump.mount`` are 
+deleted by RPM.
+
+To avoid losing the files, the :doc:`upgrade procedure </upgrade/upgrade-to-enterprise/upgrade-guide-from-5.0-to-2022.1/upgrade-guide-from-5.0-to-2022.1-rpm/>` 
+includes a step to backup the .mount files:
+
+
+.. code-block:: console
+
+    for conf in $( rpm -qc $(rpm -qa | grep scylla) | grep -v contains ) /etc/systemd/system/{var-lib-scylla,var-lib-systemd-coredump}.mount; do sudo cp -v $conf $conf.backup-4.3; done
```

Comment:
@syuu1228 why 4.3 here in `$conf.backup-4.3;` ?

Reply:
+1

or clarify this is an example, and one should use the relevant release number

Reply:
@tzach @syuu1228 I've modified that section to improve clarity. Could you re-review it?